src/script/main/app.ts

```diff
@@ -346,7 +345,6 @@ export class App {
     const platformCssClass = Runtime.isDesktopApp() ? 'platform-electron' : 'platform-web';
     document.body.classList.add(osCssClass, platformCssClass);
 
-    new ThemeViewModel(this.repository.properties);
```

Comment:
Initializing a theme should not be an `app` concern. 
The view (react component) should be the layer that is responsible for initializing those kind of things